src/Mezzurite.ts

```diff
@@ -153,9 +153,13 @@ export class MezzuritePlugIn implements ITelemetryPlugin{
                     isRedirect  = obj.value;
                 }
                 else {
+                    if (obj.value > 0) {
                     props["Redirect"] = isRedirect;
```

Comment:
isRedirect is populated in the if loop above. does it have a value in the else here?

Reply:
Yes. it's declared above it.